packages/sdk/contractkit/src/wrappers/Governance-l2.test.ts

```diff
@@ -0,0 +1,29 @@
+import { setupL2, testWithAnvil } from '@celo/dev-utils/lib/anvil-test'
+import Web3 from 'web3'
+import { newKitFromWeb3 } from '../kit'
+import { ContractVersion } from '../versions'
+
+testWithAnvil('GovernanceWrapper', (web3: Web3) => {
+  describe('Hotfixes', () => {
+    it('gets L2 hotfix record for version >= 1.5.0.0', async () => {
+      const kit = newKitFromWeb3(web3)
+      const governance = await kit.contracts.getGovernance()
+      const hotfixHash = Buffer.from('0x', 'hex')
+
+      await setupL2(web3)
```

Comment:
feels like this should be first. i guess it works as is though.

Reply:
This should work as long as the transactions you are testing happens after L2 setup